api/HTMLDataListElement.json

```diff
@@ -0,0 +1,100 @@
+{
+  "api": {
+    "HTMLDataListElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLDataListElement",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "62"
+          },
+          "chrome_android": {
+            "version_added": false
```

Comment:
I can't confirm that the Android flavors should be `false`. I'd be more comfortable with `null`. This comment applies to `options` as well.